pages/common/cat.md

```diff
@@ -10,6 +10,14 @@
 
 `cat {{file1}} {{file2}} > {{target-file}}`
 
-- Append serveral files into the target file:
+- Append several files into the target file:
 
 `cat {{file1}} {{file2}} >> {{target-file}}`
+
+- Number the non-blank lines:
```

Comment:
I don't mean that, I just thought `-n` is more common than `-b` (a specialized version of `-n` you could say).